app/assets/stylesheets/patternfly_overrides.scss

```diff
@@ -714,3 +714,13 @@ fieldset {
 table.c3-tooltip td {
   background-color: #393f44;
 }
+
+
+  #action_info_div{
```

Comment:
@himdel @skateman don't we have the similar issue in other forms as well, thinking if this should be fixed in general instead of only for one specific div, please suggest

Reply:
I think we previously solved issues like this by adding an untruncated `title` attribute.

Other than that ..maybe making all selects determine their width automatically would be quite nice.
But it could also bring breakages, esp. with old (jquery) selects.